src/frontend/screens/Library/index.css

```diff
@@ -3,7 +3,7 @@
   display: grid;
   grid-template-columns: repeat(auto-fill, minmax(156px, 1fr));
   grid-gap: 1.5rem;
-  padding: 0 var(--space-md-fixed);
+  padding: 0 var(--space-md-fixed) 7rem;
```

Comment:
Maybe create a variable for that 7rem specific to the bottom library:
`var(--bottom-space)` or something like this

Reply:
where would I create this specific variable?